ui/cap-react/src/antd/admin/components/SchemaWizard.js

```diff
@@ -18,10 +18,7 @@ const SchemaWizard = ({ field, loader }) => {
   return (
     <DndProvider backend={HTML5Backend}>
       <Row style={{ height: "100%" }} gutter={[16, 16]}>
-        <Col
-          span={5}
-          style={{ overflowX: "hidden", height: "100%", padding: "15px" }}
-        >
+        <Col span={5} style={{ overflowX: "hidden", height: "100%" }}>
```

Comment:
There is an extra scrollbar that makes no sense, we should try to avoid

![Screenshot from 2023-01-19 13-46-53](https://user-images.githubusercontent.com/6625469/213446814-84c35eb6-4f99-463d-927f-5a73d97d3b61.png)

Reply:
Which browser are you using? I don't see a double scrollbar in neither Chrome, Firefox nor Safari

Reply:
For the record, there was a useless scrollbar and in MacOS scrollbars are only displayed by default when scrolling (this is a system setting, so it happens in all browsers). After disabling the setting I could see the scrollbar and it is now fixed.